config/dispatcher/dispatcher.ex

```diff
@@ -642,7 +642,7 @@ defmodule Dispatcher do
   end
 
   match "/worship-services/*path" do
-    forward conn, path, "http://resource/worship-services/"
+    forward conn, path, "http://cache/worship-services/"
```

Comment:
literally no idea why it was in resource in the first place. so :+1:

Reply:
[This comment](https://github.com/lblod/app-digitaal-loket/pull/371#issuecomment-1479898750) explains it a bit. It was copied that way from OP but I don't think it's needed for loket.